src/main/java/software/bernie/techarium/integration/hwyla/HwylaProvider.java

```diff
@@ -0,0 +1,32 @@
+package software.bernie.techarium.integration.hwyla;
+
+import mcp.mobius.waila.api.IServerDataProvider;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.World;
+import software.bernie.techarium.tile.base.MachineMasterTile;
+import software.bernie.techarium.tile.base.MachineSlaveTile;
+
+public class HwylaProvider implements IServerDataProvider<TileEntity> {
```

Comment:
Should probably be named something else if it provides TOP data as well as hwyla

Reply:
it's unused and did only stuff for hwyla